pkg/kubefed2/federate/enable.go

```diff
@@ -78,6 +84,7 @@ func (o *enableTypeOptions) Bind(flags *pflag.FlagSet) {
 	flags.StringVar(&o.rawOverridePaths, "override-paths", "", "A common-separated list of dot-separated paths (e.g. spec.completions,spec.parallelism).")
 	flags.StringVar(&o.primitiveGroup, "primitive-group", "primitives.federation.k8s.io", "The name of the API group to use for generated federation primitives.")
 	flags.StringVar(&o.primitiveVersion, "primitive-version", "v1alpha1", "The API version to use for generated federation primitives.")
+	flags.BoolVar(&o.outputYAML, "output-yaml", false, "Whether to output yaml to stdout for the resources that will be created in the API.")
```

Comment:
I see that `json` is used as an intermediate structure; would it make sense to have an option similar to kubectl -> `-o json|yaml`